content/release-notes/2022-07-19.md

```diff
@@ -4,10 +4,10 @@ label: 'Storage'
 
 ### What's new
 
-- Safekeeper: Added support for backing up WAL to S3 for disaster recovery.
-- Safekeeper: Added support for downloading WAL from S3 on demand.
-- Safekeeper: Switched to etcd subscriptions to keep pageservers up to date regarding safekeeper status.
-- Safekeeper: Implemented JWT authentication in the Safekeeper HTTP API.
+- Safekeeper: Added support for backing up Write-ahead Logs (WAL) to S3 storage for disaster recovery.
+- Safekeeper: Added support for downloading Write-ahead Logs (WAL) from S3 storage on demand.
```

Comment:
I think we can drop the second extended name for WAL as we describe just the line before

Also official postgres docs word it as Write-**A**head Log -- with a capital A

I think we can use postgres docs as a reference for all such naming, abbreviation styles and so on

Reply:
Fixed!